src/pydata_sphinx_theme/theme/pydata_sphinx_theme/theme.conf

```diff
@@ -49,6 +49,7 @@ primary_sidebar_end = sidebar-ethical-ads.html
 footer_start = copyright.html, sphinx-version.html
 footer_end = theme-version.html
 secondary_sidebar_items = page-toc.html, edit-this-page.html, sourcelink.html
+show_version_warning_banner = False
```

Comment:
Small comments why making the default value to `False` ? I think it would be bette rif this starts to work out of the box for people making the update. What is you reasoning for deactivate it ?

Reply:
motivation is backward compatibility / no surprises.  I don't feel strongly about it though.